agent/stats/engine.go

```diff
@@ -551,6 +561,23 @@ func (engine *DockerStatsEngine) getTaskHealthUnsafe(taskARN string) *ecstcs.Tas
 			HealthStatus:  aws.String(healthInfo.Status.BackendStatus()),
 			StatusSince:   aws.Time(healthInfo.Since.UTC()),
 		}
+
+		// Get status message for firelens v2 container
+		firelensConfig := dockerContainer.Container.GetFirelensConfig()
+		if firelensConfig != nil && firelensConfig.Version == "v2" && firelensConfig.StatusMessageReportingPath != "" {
+			var statusMessage string
+			StatusMessageFilepath, err := getStatusMessageReportingPath(engine.config.DataDirOnHost, taskARN, dockerContainer.Container.Name)
+			switch {
+			case err != nil:
+				seelog.Errorf("Failed to %s: %v", taskARN, err)
+			case StatusMessageFilepath == "":
+				seelog.Debugf("Status message filepath is empty")
```

Comment:
From here it seems when `StatusMessageFilepath` is empty the `StatusMessage` will be set to be empty as well. Is it desired? Why it is not an error? If this is an expected case, when would `StatusMessageFilepath` is expected to be empty and